docs/docs/02-concepts/05-simulator.md

```diff
@@ -94,6 +94,34 @@ const response = await fn.invoke("hello!");
 console.log(response);
 ```
 
+### Interacting with the simulation across processes
```

Comment:
(disclaimer) Note that earlier in this doc it says these APIs are subject to breaking changes. So take with a grain of salt...